packages/node/src/utils/index.ts

```diff
@@ -9,11 +9,11 @@ export function arraify<T>(value: T | T[]): T[] {
   return Array.isArray(value) ? value : [value]
 }
 
-export function unimplemented(info?: string): never {
```

Comment:
never would actually be the correct type for this function.

It doesn't ever return a value since it always throws in every condition.

Reply:
This.